lib/kwtool_web/router.ex

```diff
@@ -21,6 +21,7 @@ defmodule KwtoolWeb.Router do
 
     get "/", PageController, :index
     get "/sign_up", AuthController, :show
+    get "/sign_in", AuthController, :index
```

Comment:
Technically, sign in should be `:new` as we create a new session 💭

Reply:
yes, it seems to make more sense. Updated!